src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -81,7 +100,27 @@ public static MythicMob getMythicMob(String name) {
         return MythicMobs.inst().getMobManager().getMythicMob(name);
     }
 
-    public static Entity spawnMythicMob(MythicMob mythicMob, Location location, int level) {
-        return mythicMob.spawn(BukkitAdapter.adapt(location), level).getEntity().getBukkitEntity();
+    public static MobManager getMobManager() {
+        return MythicMobs.inst().getMobManager();
+    }
+
+    public static SpawnerManager getSpawnerManager() {
+        return MythicMobs.inst().getSpawnerManager();
+    }
+
+    public static boolean isMythicSpawner(String name) {
+        return (!(MythicMobs.inst().getSpawnerManager().getSpawnerByName(name) == null));
```

Comment:
... is this mess of parens and symbols meant to just be `!=`?

Reply:
Yea......